packages/react-app/src/pages/manage.tsx

```diff
@@ -1,12 +1,14 @@
+import './manage.scss';
```

Comment:
Instead of importing multiple SCSS files, let's have an index SCSS file that imports all the SCSS files and import that on the App.js file. That way it is easier to manage all files.